src/scenes/AcceleratorRouter/Participants/ParticipantsView.tsx

```diff
@@ -106,16 +100,6 @@ export default function ParticipantsView(): JSX.Element {
 
     return (
       <Box display='flex' justifyContent='right'>
-        {canUpdateParticipants && (
```

Comment:
Are we removing the edit button entirely?

Reply:
Should AcceleratorAdmin+ be able to edit participants?

Reply:
Oh, you are right! We need to allow AcceleratorAdmin+ to edit. Will fix, thanks